src/models/SqlOutputContentProvider.ts

```diff
@@ -314,16 +314,38 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
         // Always run this command even if just updating to avoid a bug - tfs 8686842
 
         // Find any URIs which match the one we are about to display
-        let resultPaneURIMatch = vscode.workspace.textDocuments.find(tDoc => tDoc.uri.toString() === resultsUri);
+        let resultPaneTextDocument = vscode.workspace.textDocuments.find(tDoc => tDoc.uri.toString() === resultsUri);
```

Comment:
Raymond is checking in a change to this same method - I'd recommend merging his changes in and then applying your fix.